datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/db.yaml

```diff
@@ -25,14 +25,37 @@
                  The number of columns must equal the number of columns returned in the query.
                  There are 2 required pieces of data:
                    a. name - The suffix to append to `<INTEGRATION>.` to form
-                             the full metric name. If `type` is `tag`, this column is
+                             the full metric name. If `type` is a `tag` type, this column is
                              considered a tag and applied to every
                              metric collected by this particular query.
                    b. type - The submission method (gauge, monotonic_count, etc.).
-                             This can also be set to `tag` to tag each metric in the row
-                             with the name and value of the item in this column. You can
-                             use the `count` type to perform aggregation for queries that
-                             return multiple rows with the same or no tags.
+                             This can also be set to the following `tag` types to
+                             tag each metric in the row with the name and value
+                             of the item in this column:
+                              a. tag          - This is the default tag type
```

Comment:
Can you use a different alphanumeric like i. ii. ? Using the same lettering for nested bullets might be confusing to the reader.